src/Hydration.js

```diff
@@ -1,30 +1,27 @@
 class Hydration {
-  constructor(data) {
-    this.data = data;
-    this.userHydrationData = [];
+  constructor(hydrationData) {
+    this.hydrationData = hydrationData;
   }
 
   getUserHydrationData(id) {
-    this.userHydrationData = this.data.filter(data => {
-      return data.userID === id
-    })
+    return helper.getUserData(this.hydrationData, id);
```

Comment:
How is this file accessing our helper fx? Does that occur within the tests or scripts js? Reading as is makes me think we would get an error of undefined.

Reply:
I am exporting our helper.js file in HTML script before the script.js so that the helper file can access all data needed.